src/utils/languages.js

```diff
@@ -1486,3 +1486,11 @@ export const setAppLanguage = lang => {
 export const getAppLanguage = () => {
   return localStorage.getItem('language-app')
 }
+
+export const setSearchLanguage = lang => {
+  localStorage.setItem('language-search', lang)
+}
+
+export const getSearchLanguage = () => {
+  return localStorage.getItem('language-search')
```

Comment:
```suggestion
  return localStorage.getItem('language-search') || 'en'
```

default search english when there cannot found from localStorage?

Reply:
I like this idea. I had done this in `Search` thinking along the same lines (mimicking `src/index`):

```
  const lang = getSearchLanguage() || navigator.language.substr(0, 2)
```

Do you think it makes sense to have both?

Reply:
Nope, as `getSearchLanguage()` already returns value, there is no way to get the second value.

Reply:
Will opt for your suggestion then and get rid of `|| navigator.language.substr(0, 2)`

Reply:
And actually before committing: why wouldn't we be doing the same thing with `getAppLanguage`? Do we need to worry about staying consistent overall with these methods?

Reply:
yup, i think yes, when we remove the changing for app language page, there is no place to set app language in localStorage

Q : index.js `const lang = getAppLanguage() || navigator.language.substr(0, 2)`

what if there isn't `getAppLanguage` and the navigator language is not supported in our language json (assume the edge case)? should we set default to `en`?